ddl/db_test.go

```diff
@@ -2020,23 +2020,18 @@ func (s *testDBSuite3) TestGeneratedColumnDDL(c *C) {
 	s.tk.MustExec("use test")
 
 	// Check create table with virtual generated column.
```

Comment:
```suggestion
	// Check create table with virtual and stored generated columns.
```
Comment should be updated as well.